lib/proxy.js

```diff
@@ -137,12 +137,13 @@ function buildResultingMiddleware (host, options, stream) {
     var reqOpts = {
       url     :  URL.format(proxyUrl),
       method  : req.method,
-      headers : _.defaults(options.headers || {}, req.headers),
+      // headers : _.defaults(options.headers || {}, req.headers),
+      headers : options.headers,
```

Comment:
I agree.

What is your goal with this though? If the user sends a request from a browser to this endpoint and they don't have headers that an API agrees with should they fix the headers in the browser request? or should the proxy handle headers to the API?